packages/design-system/src/components/snackbar/stories/index.js

```diff
@@ -52,154 +51,144 @@ const Container = styled.div`
 
 export default {
   title: 'DesignSystem/Components/Snackbar',
+  args: {
+    actionLabel: '',
+    customZIndex: 1,
+    dismissible: true,
+    preventAutoDismiss: true,
+    message: 'Sorry! File failed to upload.',
+  },
 };
 
-export const _default = () => (
+export const _default = (args) => (
   <ThemeProvider theme={theme}>
     <Container>
       <SnackbarContainer
         notifications={[
           {
-            actionLabel: text('actionLabel', ''),
-            'aria-label': text(
-              'aria-label',
-              'this is my aria label giving my message context for screen reader users'
-            ),
-            customZIndex: number('customZIndex'),
-            dismissible: boolean('dismissible'),
-            preventAutoDismiss: boolean('preventAutoDismiss'),
-            message: text('message', 'Sorry! File failed to upload.'),
+            'aria-label':
+              'this is my aria label giving my message context for screen reader users',
             onAction: action('on action clicked'),
             onDismiss: action('on dismiss fired'),
+            ...args,
```

Comment:
💭 we could cut down most of the stories here because the controls give us the capability of adding/removing different things in the snackbar (actions and text)

![customize-errythang](https://user-images.githubusercontent.com/22185279/151244691-ad09a165-70cb-44e9-9906-bfb6faab7e03.gif)

Reply:
Could remove most of the snackbar stories but that seems like a ticket for a different time